src/zope/apidoc/presentation.py

```diff
@@ -124,7 +124,7 @@ def getViews(iface, type=IRequest):
     for reg in gsm.registeredAdapters():
         if (len(reg.required) > 0 and
             reg.required[-1] is not None and
-            reg.required[-1].isOrExtends(type)):
+                reg.required[-1].isOrExtends(type)):
```

Comment:
Indent the line above too, for consistency?

Reply:
Nice catch.